hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobsResource.java

```diff
@@ -284,7 +284,9 @@ public Response inspect (
       required = true)  @QueryParam("path") String path,
     @Context HttpServletRequest req) throws JobException {
     Users user = jWTHelper.getUserPrincipal(req);
-    JobConfiguration config = jobController.inspectProgram(path, project, user, jobtype);
+    JobConfiguration config =
+      jobController.inspectProgram("/" + Settings.DIR_ROOT + "/" + project.getName() + "/" + path
```

Comment:
Nitpick: there is an util function in `Utils.java` that assembles the project path